### PR TITLE
Fail github actions on mvn verify failure

### PR DESCRIPTION
**Pull request by sfc-gh-japatel**
- Remove travis related code.

### CODE REVIEW

File: src/test/java/net/snowflake/ingest/SimpleIngestIT.java [hunk 1]
```diff
@@ -233,7 +233,7 @@ private void getHistoryAndAssertLoad(SimpleIngestManager manager, String test_fi
     //try to wait until the future is done
     try {
       //wait up to 2 minutes to load
```
Comment (sfc-gh-rramachandran) [inline src/test/java/net/snowflake/ingest/SimpleIngestIT.java:235]: NIT: Fix comment. Is it taking more than 2 minutes for the load to complete? Should we reduce the number of files?

File: .github/workflows/End2EndTest.yml [hunk 2]
```diff
@@ -30,5 +30,6 @@ jobs:
         env:
           JACOCO_COVERAGE: true
           WHITESOURCE_API_KEY: ${{ secrets.WHITESOURCE_API_KEY }}
+        continue-on-error: false
```
Comment (sfc-gh-rramachandran) [inline .github/workflows/End2EndTest.yml:33]: Surprised this is not the default behavior :)

**Review by sfc-gh-rramachandran — APPROVED** [review]
LGTM!

File: src/test/java/net/snowflake/ingest/SimpleIngestIT.java (hunk 1 above)
Comment (sfc-gh-japatel) [reply src/test/java/net/snowflake/ingest/SimpleIngestIT.java:235]: Will fix. We are just adding 1 file, shouldnt take too long. I will revert to 2 mins.

**Pull request merged**